hedera-node/hedera-mono-service/src/test/java/com/hedera/services/store/contracts/precompile/ERC20PrecompilesTest.java

```diff
@@ -1701,6 +1703,10 @@ private void givenPricingUtilsContext() {
                             new SyntheticTxnFactory.FungibleTokenTransfer(
                                     -AMOUNT, false, token, sender, null)));
 
+    public static final CryptoTransferWrapper CRYPTO_TRANSFER_TOKEN_WRAPPER =
+            new CryptoTransferWrapper(
+                    new TransferWrapper(Collections.emptyList()),
+                    Collections.singletonList(TOKEN_TRANSFER_WRAPPER));
```

Comment:
Can't we directly use the new constants defined in `HTSTestsUtil`, removing the need for the new ones here?

Reply:
done